tools/bots/mission-control-bot.ts

```diff
@@ -0,0 +1,202 @@
+import { Client, MessageEmbed, Message } from "discord.js";
+import Web3 from "web3";
+
+
+const TOKEN_DECIMAL = 18n;
+const FAUCET_SEND_INTERVAL = 1; // hours
+const EMBED_COLOR_CORRECT = 0x642f95;
+const EMBED_COLOR_ERROR = 0xc0392b;
+
+const params = {
+	// Discord app information
+	DISCORD_TOKEN: process.env.DISCORD_TOKEN,
+	DISCORD_CHANNEL: process.env.DISCORD_CHANNEL,
+
+	// Web3 RPC access
+	RPC_URL: process.env.RPC_URL,
+	ACCOUNT_KEY: process.env.ACCOUNT_KEY,
+
+	// Token distribution
+	TOKEN_COUNT: BigInt(process.env.TOKEN_COUNT || 10),
+}
+
+Object.keys(params).forEach(param => {
+	if (!params[param]) {
+		console.log(`Missing ${param} env variables`);
+		process.exit(1);
+	}
+})
+
+const web3Api = new Web3(params.RPC_URL);
+
+console.log(`Starting bot...`);
+console.log(`Connecting web3 to ${params.RPC_URL}...`);
+
+const client: Client = new Client();
+const receivers: { [author: string]: number } = {};
+
+client.on("ready", () => {
+	console.log(`Logged in as ${client.user.tag}!`);
+});
+
+/**
+ * Returns the approximated remaining time until being able to request tokens again.
+ * @param {Date} lastTokenRequestMoment Last moment in which the user requested funds
```

Comment:
shouldn't be the type {number} ?

Reply:
Yep, my bad. I changed the type of the parameter in the function definition, but I forgot the annotation. Should I submit a new PR just for this?